File: LeetCode/1365有多少小于当前数字的数字.cpp
/**
 * Note: The returned array must be malloced, assume caller calls free().
 */
int* smallerNumbersThanCurrent(int* nums, int numsSize, int* returnSize){
    int *res = (int *)malloc(sizeof(int) * numsSize);
    *returnSize = 0;
    for (int i = 0; i < numsSize; i++) {
        int cnt = 0;
        for (int j = 0; j < numsSize; j++) {
            if (j != i && nums[j] < nums[i]) cnt++;
        }
        res[(*returnSize)++] = cnt;
    }
    return res;
}

File: LeetCode/1365有多少小于当前数字的数字.go
func smallerNumbersThanCurrent(nums []int) []int {
    var ans = make([]int, len(nums))
    cnt := 0
    for k, v := range nums {
        cnt = 0
        for k1, v1 := range nums {
            if k1 != k && v > v1 {
                cnt++
            }
        }
        ans[k] = cnt
    }
    return ans
}